src/components/tab-title/tab-title.e2e.ts

```diff
@@ -50,6 +84,241 @@ describe("calcite-tab-title", () => {
     expect(iconEnd).not.toBeNull();
   });
 
+  describe("basic closing behavior", () => {
+    let page: E2EPage;
+
+    beforeEach(async () => {
+      page = await newE2EPage();
+      await page.setContent(tabTitleClosableHtml);
+    });
+
+    it("clicking on close button closes the tab", async () => {
+      const close = await page.find(closeHtml);
+
+      await close.click();
+      await page.waitForChanges();
+
+      const containerEl = await page.find(`calcite-tab-title >>> .${CSS.container}`);
+      expect(await containerEl.getProperty("hidden")).toBe(true);
+    });
+
+    it("becomes no longer closable when it's the last remaining tab", async () => {
+      page = await newE2EPage();
+      await page.setContent(
+        html`
+          <tab-nav>
+            <calcite-tab-title id="one" closable>Text</calcite-tab-title>
+            <calcite-tab-title id="two" closable>Text</calcite-tab-title>
+          </tab-nav>
+        `
+      );
+
+      let containerElOne = await page.find(`calcite-tab-title[id='one']`);
+      const closeOne = await page.find(`calcite-tab-title[id='one'] >>> .${CSS.closeButton}`);
+      expect(containerElOne).toHaveAttribute(HYDRATED_ATTR);
+
+      await closeOne.click();
+      await page.waitForChanges();
+
+      containerElOne = await page.find(`calcite-tab-title[id='one']>>> .${CSS.container}`);
+      expect(await containerElOne.getProperty("hidden")).toBe(true);
+
+      const closeTwo = await page.find(`calcite-tab-title[id='two'] >>> .${CSS.closeButton}`);
+      expect(await closeTwo.getProperty("closable")).not.toBe(true);
+    });
+  });
+
+  describe("closing sequence", () => {
+    let page: E2EPage;
+
+    let matchingTabEl: E2EElement;
+    let close: E2EElement;
+
+    const closeTabsInSequenceOfGivenArrayOfIds = async (arrayOfIds: string[]): Promise<void> => {
```

Comment:
Let's make this a function that takes a page as an argument as well as an array of IDs.

Also, can we rename `arrayOfIds` to be more expressive? Once this is done, I think we could drop `OfGivenArrayOfIds` from the function name as the function signature provides context.